lib/util/common.ts

```diff
@@ -59,3 +59,7 @@ export function detectPlatform(
 
   return null;
 }
+
+export function noLeadingAtSymbol(input: string): string {
+  return input.length && input.startsWith('@') ? input.slice(1) : input;
```

Comment:
@rarkins @secustor wonder why we do a length check here. it doesn't catch or improve anything here.
```suggestion
  return input.startsWith('@') ? input.slice(1) : input;
```

Reply:
Is this copied from somewhere?

Reply:
yes, code was moved from worker below

Reply:
Then 🤷‍♂️